src/mpid/ch4/shm/ipc/src/ipc_win.c

```diff
@@ -167,7 +167,8 @@ int MPIDI_IPC_mpi_win_create_hook(MPIR_Win * win)
                 case MPIDI_IPCI_TYPE__GPU:
                     mpi_errno =
                         MPIDI_GPU_attach_mem(ipc_shared_table[i].mem_handle.gpu,
-                                             attr.gpu_attr.device, &shared_table[i].shm_base_addr);
+                                             attr.gpu_attr.device, MPI_INT,
```

Comment:
The window is treated as an array of bytes (or some other type based on the displacement unit).  The usage of `MPI_INT` seems pretty arbitrary here.